L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -882,14 +885,15 @@ namespace trklet {
     // This is a temporary fix for compatibilty with HLS. We will need to implement multiple match
     // printing in emulator eventually, possibly after CMSSW-integration inspired rewrites
     // Use false when generating HLS files, use true when doing full hybrid tracking
-    bool doMultipleMatches_{true};
+    bool doMultipleMatches_{false};
```

Comment:
Settings.h should use the ultimate values, as the trigger group wants this. The doc https://github.com/cms-L1TK/firmware-hls/blob/master/README.md records which of these need changing when generating test data to match the current FW.